pkg/egressip/egressip_test.go

```diff
@@ -0,0 +1,403 @@
+package egressip
```

Comment:
I don't see tests for Delete and Exists, could you add them please?

Reply:
done